src/styles/product-card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 198px;
+  height: 374px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  color: $main;
+  padding-block: 32px 0;
+
+  &__image,
+  &__descreption,
+  &__code,
+  &__reviews,
+  &__price,
+  &__button {
+    width: 166px;
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+  }
```

Comment:
This styles are redundant, please remove them.

Reply:
Thanks, solved.